packages/nextjs/src/config/webpack.ts

```diff
@@ -695,7 +695,7 @@ export function getWebpackPluginOptions(
   const defaultPluginOptions = dropUndefinedKeys({
     include: isServer ? serverInclude : clientInclude,
     ignore:
-      isServer || !userSentryOptions.widenClientFileUpload
+      isServer || userSentryOptions.widenClientFileUpload
```

Comment:
It seems that this feature has been broken since it was implemented...

https://github.com/getsentry/sentry-javascript/pull/4705/files#diff-55396f0a8f12ba2eb7ce27f3cf886f946c29c2aa93b5d2f993dadbd5a9f3cc75